app/src/main/java/com/mapbox/maps/testapp/examples/DistanceExpressionActivity.kt

```diff
@@ -24,15 +23,16 @@ class DistanceExpressionActivity : AppCompatActivity() {
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.activity_within_expression)
+    val binding = ActivitySimpleMapBinding.inflate(layoutInflater)
```

Comment:
Why not using `DistanceExpressionActivityBinding`?

Reply:
I've deleted all xml layouts that actually duplicate `activity_simple_map` and replaced.

Reply:
imo we should have separate layout for different activities.

Reply:
It's confusing when reading the source code, imho it's better to make our examples more self contained so it's easier to follow by developers.